Av1an/chunk_queue.py

```diff
@@ -127,7 +127,7 @@ def create_vsffms2_chunk(args: Args, index: int, load_script: Path, frame_start:
     frames = frame_end - frame_start
     frame_end -= 1  # the frame end boundary is actually a frame that should be included in the next chunk
 
-    ffmpeg_gen_cmd = f'vspipe {load_script} -y - -s {frame_start} -e {frame_end}'
+    ffmpeg_gen_cmd = ('vspipe', load_script, '-y', '-', '-s', frame_start, '-e', frame_end)
```

Comment:
Some of these are not strings and now need to be explicitly converted to strings.

```suggestion
    ffmpeg_gen_cmd = ('vspipe', load_script.as_posix(), '-y', '-', '-s', str(frame_start), '-e', str(frame_end))
```